{{ cookiecutter.repo_name }}/.flake8

```diff
@@ -4,9 +4,9 @@ exclude =
     __pycache__,
     build,
     dist,
-    versioneer.py,
-    {{ cookiecutter.package_dir_name }}/_version.py,
+    src/diffpy/{{ cookiecutter.package_dir_name }}/version.py,
```

Comment:
Did we decide that we would make a separate cookie cutter for `diffpy` packages vs non-diffpy?  I am ok either way.  It is easier to build the cookiecutter if we don't have conditionals in there, but then the downside is that we have to maintain multiple cookie cutters if we have one for each type of package.